cmd/apiserver/drun_simulator.go

```diff
@@ -0,0 +1,179 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"math/rand"
+	"time"
+
+	"github.com/sirupsen/logrus"
```

Comment:
one last thing. Rename this file to `simulator.go` and fix the order of the imports.

Reply:
Don't you have 5 minutes to explain why do we need to do it like this on a call?

Reply:
This is an idiomatic Go principal.